### PR TITLE
Course creation flow

### PR DESCRIPTION
**Pull request by OlgaLopushanska**
merged crud entities

### CODE REVIEW

File: src/main/java/com/itacademy/cms/repository/UserToCourseDAO.java [hunk 1]
```diff
@@ -0,0 +1,6 @@
+//package com.itacademy.cms.dao;
```
Comment (dmakhun) [inline src/main/java/com/itacademy/cms/repository/UserToCourseDAO.java:1]: please delete this and all commented code

File: src/main/java/com/itacademy/cms/repository/CategoryDAO.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package com.itacademy.cms.repository;
+
+import com.itacademy.cms.model.Category;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface CategoryDAO extends CrudRepository<Category, Long> {
```
[the remark this answers is not in the document]
Comment (OlgaLopushanska) [reply src/main/java/com/itacademy/cms/repository/CategoryDAO.java:9]: thanks I did it

File: src/main/java/com/itacademy/cms/repository/UserToCourseDAO.java (hunk 1 above)
Comment (OlgaLopushanska) [reply src/main/java/com/itacademy/cms/repository/UserToCourseDAO.java:1]: ok I did it

**Comment by OlgaLopushanska** [comment]
I`ll make some changes

**Pull request closed without merging**